charts/wandb-base/templates/service.yaml

```diff
@@ -30,6 +30,9 @@ metadata:
     cloud.google.com/backend-config: '{"default": "{{ include "wandb-base.fullname" . }}-backend-config"}'
     {{- end }}
 spec:
+  {{- if .Values.service.headless }}
```

Comment:
should we allow a headless and non-headless service to exist simultaneously

Reply:
or always create a normal service and optionally create the second headless service name <service_name>-headless

Reply:
The latter is better. I'll make the change.

Reply:
i guess we can always change this when the need comes up

Reply:
oh, didnt see youd already replied

Reply:
@danielpanzella updated. Does this look ok? Not sure if the right thing is to exactly copy the service attributes or pull from `service.headless`. WDYT?